wire-ios-system/Source/WireLogger.swift

```diff
@@ -136,4 +148,5 @@ public protocol LoggerProtocol {
   func error(_ message: String, attributes: LogAttributes?)
   func critical(_ message: String, attributes: LogAttributes?)
 
+  func prepareMessage<Payload: LoggablePayload>(title: String, payload: Payload) -> String
```

Comment:
I don't quite understand what you mean.

Reply:
It is `LoggablePayload` not `LoggableProtocol` :) So that any payload could type could be passed here and transformed to string. However in the upgrade you've suggested we could move encoding to `logDescription` of `SharingLog` that would implement `LogConvertible`.